hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/util/impl/UtilPreTransactionHandlerImpl.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.service.util.impl;
+
+import com.hedera.node.app.SigTransactionMetadata;
+import com.hedera.node.app.service.token.impl.AccountStore;
+import com.hedera.node.app.service.util.UtilPreTransactionHandler;
+import com.hedera.node.app.spi.meta.TransactionMetadata;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+
+/**
+ * A {@code UtilPreTransactionHandler} implementation that pre-computes the required signing keys
```

Comment:
I think I like using "@link" here instead of "@code" because when you refactor names, or if you cmd+click, it recognizes the name as an actual class name and is involved in refactoring / navigation.